formal-spec/measurements.tex

```diff
@@ -174,6 +177,102 @@ \section{Impact on performance}
 %% the section above.
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+\subsection{Processing Time and Memory Consumption}
```

Comment:
Please wrap everything to 80 chars :pray:

Reply:
OK